Documentation/Signoffs/jdjoyce42-power-signoff.md

```diff
@@ -0,0 +1,58 @@
+# Power Subsystem
+
+
+  
+ 
+## Function of the Subsystem
+
+The purpose of this subsystem is to take the voltage supplied from a wall outlet and make it usuable for the project. Every component of our project requires our circuit to be DC. So we have reviewed ways to implement this as efficiently as possible. It is more efficient for us to use a wall wart for our project. Wall warts have seen widespread use in the last ten or so years due to their low cost, ease of manufacture, versatility, and sonic improvements; equipment with external power supplies are usually quieter (all other things being equal) than equipment with internal supplies.We have looked into building our own circuit on a breadboard and just connecting it to the rest of the project, but that causes us to not meet the constraint of it looking clean and not distracting as opposed to simply having a wall wart. The wall wart is essentially doing what is shown in Figure 1. The device will take in the AC current from the wall and convert it to DC so it can work properly with the rest of our devices.
+
+
+## Constraints
```

Comment:
Ambiguous. How do these constrain you?

Reply:
Some of these seem to be duplicates? Also, why is there a temperature requirement? If it is appropriate here, why is it not appropriate everywhere else?

Reply:
No standards?